src/status_im/contexts/wallet/common/utils.cljs

```diff
@@ -275,7 +275,7 @@
     {:token               (:symbol token)
      :token-name          (:name token)
      :state               :default
-     :metrics?            true
+     :metrics?            (money/has-funds? balance)
```

Comment:
When I'm reading `has-funds?` I have a feeling that it is some calculation on complex big structure whereas it is simply a comparison to zero :) I'd suggest to rename `has-funds?`-> `greater-than-zero?` to leave `utils.money` on lower level of abstraction, wdyt?

Reply:
Good point! Renamed it to `above-zero?` (basically the same but shorter) and replaced the usage of `greater-than` where the value is compared to `0` to use this function instead.